extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeDashManifestCreatorTest.java

```diff
@@ -0,0 +1,44 @@
+package org.schabi.newpipe.extractor.services.youtube;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.schabi.newpipe.downloader.DownloaderTestImpl;
+import org.schabi.newpipe.extractor.NewPipe;
+import org.schabi.newpipe.extractor.exceptions.ExtractionException;
+
+import java.io.IOException;
+
+public class YoutubeDashManifestCreatorTest {
+    @Before
+    public void setup() throws IOException {
+        NewPipe.init(DownloaderTestImpl.getInstance());
+    }
+
+    @Ignore
+    @Test
+    public void testOtfStream() throws ExtractionException,
+            YoutubeDashManifestCreator.YoutubeDashManifestCreationException {
+        final String otfUrl = "";
+        final ItagItem itagItem1 = ItagItem.getItag(247);
+        final String mimeType = "video/webm; codecs=\"vp9\"";
+        itagItem1.setCodec(mimeType.split("\"")[1]);
+        itagItem1.setBitrate(1505280);
+        itagItem1.setQuality("hd720");
+        itagItem1.fps = 25;
+        itagItem1.setWidth(1280);
+        itagItem1.setHeight(720);
+        System.out.println(YoutubeDashManifestCreator.createDashManifestFromOtfStreamingUrl(otfUrl, itagItem1));
```

Comment:
Mmmh, wouldn't this spam the gradle test output with a full dash manifest? Wouldn't it be better if you checked whether it contains something?

Reply:
Yes, it was a test to quickly check if the generation was correct. I will write a better test.